warehousing-alternatives-compare-table.md

```diff
@@ -0,0 +1,18 @@
+## Differences between Cloudant to Db2 data flow solutions
+
+### Source options from Cloudant:
+
+technology | available document extract endpoints | notes
+--- | --- | --- 
+Apache Bahir Spark connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
+IBM Cloud Functions | `/{db}/_changes`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | - 
+Node-RED | `/{db}/_design/{design_doc}/_search/{search_index_name}` | -
+
+### Sink options to Db2:
+
+technology | available SQL operations for writing to Db2 | notes
+--- | --- | ---
+Apache Bahir Spark connector | CREATE TABLE, INSERT | Spark save modes change table creation behavior:<br/>`SaveMode.ErrorIfExists` - try to create the table and error if it exists.<br/>`SaveMode.Append` - try to create the table, insert rows into existing table if present.<br/>`SaveMode.Overwrite` - try to create the table, dropping it first if it already exists.<br/>`SaveMode.Ignore` - try to create the table, doing nothing if it already exists.
+IBM Cloud Functions | ANY | SQL operations available via bindings in multiple languages.
```

Comment:
I think any should be lowercase (even though it won't be monospaced) to distinguish it from the SQL commands

Reply:
Fixed in c35a74c.